docs/core/development/review-process.rst

```diff
@@ -0,0 +1,248 @@
+  * **If you're filing a bug because you noticed a regression** in the current pre-release,
+    make sure to mark it as a regression,
+    add it to the release milestone, and to post something to the mailing list as well.
+    It's very important for us to spot regressions before they make it into a final release,
+    so redundant communication is better in this case!
+  * Try **not** to *write too much code at once*.
+    A good change needs to be under 1000 lines of diff for a reviewer to be able to deal with it before getting fatigued and missing things.
+    You should be aiming for things in the neighborhood of 200 lines.
+    This is for your benefit as well as the reviewer's; we don't want you to spend weeks on a 10,000-line monster of a feature,
+    only to have it rejected because we're not interested in having such a feature at all.
+
+Authors: Things your branch or patch must contain
+-------------------------------------------------
+
+ * Code which follows the :doc:`coding standards </core/development/policy/coding-standard>`.
+ * 100% unit test coverage for all modified and new code (even if it didn't have tests before)
+ * 100% API docstring coverage for all modified and new code (even if it didn't have docs before)
+ * No :doc:`backwards-incompatible </core/development/policy/compatibility-policy>` changes.
+   Also be sparing when adding 'public' names to the API, as they must be supported in the future.
+   If it can start with an underscore and not be exposed publicly, it probably should.
+ * Appropriate new or modified "End User" guide documentation (in the form of rst files in the `docs/ <https://github.com/twisted/twisted/tree/trunk/docs>`_ directory)
+ * A file in all relevant "newsfragments" directories describing changes which affect users See the Newsfiles section below.
+
+
+Authors: How to get your change reviewed
+----------------------------------------
+
+ * There must be an issue in the Twisted Github Issue tracker describing the desired outcome.
+   See more info below.
+   * **Note**: For security issues, see :doc:`Security </security>`.
+ * Create a GitHub Pull request
+ * Write a comment or PR description with any relevant information for the reviewer.
+ * Once the PR is ready for review,
+   leave a separate comment on that PR containing the text `please review`.
+   This will trigger the review process and will notify the review team.
+
+
+Reviewers: How to review a change
+---------------------------------
+
+ * Be familiar with the Twisted codebase, coding standard, and these review requirements.
+ * Don't be the author of the change.
+   It makes no sense to review your code, and we assume that before pushing the code to a public PR,
+   you already did the first round of self-review.
+ * Make sure that all checks are **green**!
+ * Note any unreliable / flaky tests should have a separate issue created.
+ * Assign the ticket to yourself.
```

Comment:
This is obsolete; it was designed to prevent race conditions where multiple people were reviewing at once.

```suggestion
```